this type is used to hold fds, and is passed around as such
it's wrong for it to be int.
--
diff --git a/lib/libsacd/sacd_input.c b/lib/libsacd/sacd_input.c
index d96b968..858ce33 100644
--- a/lib/libsacd/sacd_input.c
+++ b/lib/libsacd/sacd_input.c
@@ -62,7 +62,7 @@ uint32_t     (*sacd_input_total_sectors)(sacd_input_t);
 
 struct sacd_input_s
 {
-    void*              fd;
+    int              fd;
     uint8_t            *input_buffer;
     ssize_t            total_sectors;
 };
